Gruntfile.js

```diff
@@ -280,6 +280,25 @@ module.exports = function(grunt) {
                             validate: validateInputColour
                         },
                         {
+                            config: 'appConfig.kickerFont',
+                            type: 'list',
+                            default: 'egypt-regular',
+                            message: 'Kicker typeface',
+                            choices: [
```

Comment:
Hey

Something that I think would be help us to get stuff right, is to refer to the typefaces by weight number as well… So, for reference the options would be:

DE1 Display Egyptian Thin
DE2 Display Egyptian Light
DE3 Display Egyptian Regular
DE4 Display Egyptian Medium
DE6 Display Egyptian Bold

Also: this is a good selection of weights, as it covers mostly what we use, but in case DE0, DE5, and DE7 are available in the app as well, it might be good to list them as options?